package-dotorg.sh

```diff
@@ -2,6 +2,7 @@
 if [[ "$1" != "" ]]; then
     THEME="$1"
 	find $THEME/assets/sass/*.scss -type f -exec sed -i '' 's/-wpcom//g' {} \; 
+	find $THEME/assets/sass/*.scss -type f -exec sed -i '' 's/, auto-loading-homepage, jetpack-global-styles//g' {} \; 
 	find $THEME/assets/sass/*.scss -type f -exec sed -i '' 's/auto-loading-homepage, //g' {} \; 
```

Comment:
do we still need this line?

Reply:
We do but there is probably a more elegant solution.. I will think about it.

Reply:
There should be a way to do this with a regex like this:
`find $THEME/assets/sass/*.scss -type f -exec sed -i '' 's/jetpack-global-styles(,)? //g' {} \;`
sed doesn't seem to like the `?` though...

Reply:
I tried using the `-E` flag for sed to support extended regex, but couldn't figure it out.

For now I went with a solution that covers cases where the tags are not first.